src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page-content">
+    <div class="page-content__calendar">
+      <div class="calendar calendar--month-length--28">
+        <div class="calendar__day calendar__day--start-day--sun"></div>
```

Comment:
It works correctly, but `--start-day--sun` modifier should be on `calendar`, same as `--month-length--28`